# 03 Throwing-Exception-From-Function
